libwhisper/WhisperDB.cpp

```diff
@@ -0,0 +1,69 @@
+/*
+	This file is part of cpp-ethereum.
+
+	cpp-ethereum is free software: you can redistribute it and/or modify
+	it under the terms of the GNU General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	cpp-ethereum is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+	GNU General Public License for more details.
+
+	You should have received a copy of the GNU General Public License
+	along with cpp-ethereum.  If not, see <http://www.gnu.org/licenses/>.
+*/
+/** @file WhisperDB.cpp
+* @author Vladislav Gluhovsky <vlad@ethdev.com>
+* @date July 2015
+ */
+
+#include "WhisperDB.h"
+#include <boost/filesystem.hpp>
+#include <libdevcore/FileSystem.h>
+
+using namespace std;
+using namespace dev;
+using namespace dev::shh;
+
+WhisperDB::WhisperDB()
+{
+	string path = dev::getDataDir();
```

Comment:
i ahve changed to getDataDir("shh"), because it corresponds to the default value defined in function dev::getDataDir(std::string _prefix):
     if (_prefix.empty())
		_prefix = "ethereum";